lib/matplotlib/backend_bases.py

```diff
@@ -2952,6 +2952,63 @@ def _zoom_pan_handler(self, event):
             elif event.name == "button_release_event":
                 self.release_zoom(event)
 
+    def _start_event_axes_pan(self, event, method="zoom"):
+        # call "ax.start_pan(..)" on all relevant axes of an event
+
+        def _ax_filter(ax):
+            return (ax.in_axes(event) and
+                    ax.get_navigate() and
+                    getattr(ax, f"can_{method}")()
+                    )
+
+        def _capture_events(ax):
+            f = ax.get_forward_navigation_events()
+            if f == "auto":  # (capture = patch visibility)
+                f = not ax.patch.get_visible()
+            return not f
+
+        # get all relevant axes for the event
+        axes = list(filter(_ax_filter, self.canvas.figure.get_axes()))
+
+        if len(axes) == 0:
+            return
```

Comment:
```suggestion
            return []
```

Even though where this is called is gated an `if axes`, we should try to maintain type stability on the return value of this method.  Someone in 5 years will thank us.